### PR TITLE
Run QAM virtualization tests all on IPMI

### PR DESCRIPTION
**Pull request by pdostal**
Because of we cannot create multi-machine test across multiple backends I moved everything to IPMI.

- Needles: [os-autoinst-needles-sles#1093](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/1093)
- Verification run: [QEMU](http://pdostal-server.suse.cz/tests/1536)

### CODE REVIEW

**Comment by pdostal** [comment]
Hello @alice-suse, I used your [example](https://github.com/alice-suse/os-autoinst-distri-opensuse/pull/1) and it worked perfectly, thank you! You also suggested to change [os-autoinst](https://github.com/alice-suse/os-autoinst/pull/1) which I did - do you think it's necessary and if so can you open a PR please?

**Comment by alice-suse** [comment]
@pdostal Sorry for the late reply, as i was on sick leave yesterday afternoon. I am glad that https://github.com/alice-suse/os-autoinst-distri-opensuse/pull/1 works for you. 
As for os-autoinst change https://github.com/alice-suse/os-autoinst/pull/1/files, it changes the window manager display scheme for buttons on the top of window, eg maximize/minimize/close. If you think the window display in your verification example http://pdostal-server.suse.cz/tests/1536#step/virtmanager_rm_devices/26 already meets your requirement, then that os-autoinst PR is unnecessary. But if you want buttons like close in http://10.67.18.220/tests/573#step/virt_manager_example/12, then changes are needed for consoles/icewm.cfg. How do you prefer?

File: tests/virt_autotest/login_console.pm [hunk 1]
```diff
@@ -35,7 +35,12 @@ sub login_to_console {
     if (match_has_tag('prague-pxe-menu')) {
         send_key 'ret';
 
-        assert_screen([qw(grub2 grub1)], 30);
+        assert_screen([qw(grub2 grub1)], 60);
+    }
+    else {
+        # Wait for bootload for the first time.
+        $self->wait_grub(bootloader_time => 210);
+        send_key 'ret';
```
Comment (alice-suse) [inline tests/virt_autotest/login_console.pm]: @pdostal Would you please explain why this else branch needed? Actually I just removed it to let xen menuentry selection work last week with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6948/files. Now openqa xen tests fail again, see https://openqa.suse.de/tests/2534928#step/login_console/17.

Comment (alice-suse) [reply tests/virt_autotest/login_console.pm]: Fix in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7009. Please raise comments if any concern.

**Pull request merged**